src/EditorFeatures/Core/Structure/StructureTag.cs

```diff
@@ -59,6 +61,25 @@ public StructureTag(AbstractStructureTaggerProvider tagProvider, BlockSpan block
         public bool IsDefaultCollapsed { get; }
         public bool IsImplementation { get; }
 
+        public override int GetHashCode()
+            => throw ExceptionUtilities.Unreachable();
```

Comment:
This is called in editor

https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/VS-Platform?path=/src/Editor/Text/Impl/Structure/StructureSpanningTree/StructureSpanningTree.cs&version=GBmain&line=308&lineEnd=309&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

Reply:
fixed in followup.